hotspot/src/share/vm/memory/heapInspection.hpp

```diff
@@ -367,7 +367,7 @@ class HeapInspection : public StackObj {
       _csv_format(csv_format), _print_help(print_help),
       _print_class_stats(print_class_stats), _columns(columns) {}
   void heap_inspection(outputStream* st) NOT_SERVICES_RETURN;
-  size_t populate_table(KlassInfoTable* cit, BoolObjectClosure* filter = NULL) NOT_SERVICES_RETURN;
+  size_t populate_table(KlassInfoTable* cit, BoolObjectClosure* filter = NULL) NOT_SERVICES_RETURN_(0);
```

Comment:
in upstream it was fixed by the https://bugs.openjdk.org/browse/JDK-8062808:
https://github.com/openjdk/jdk8u-dev/commit/d37f26d63aeadcac6e0b39f26e0d0c2c7d0dbf71#diff-a18f898c263fcdc0d2a95dd02209ae0950811c9c995baae97726b502742f9689R370